src/main/java/seedu/classify/logic/parser/ViewStatsCommandParser.java

```diff
@@ -0,0 +1,43 @@
+package seedu.classify.logic.parser;
+
+import java.util.stream.Stream;
+
+import seedu.classify.commons.core.Messages;
+import seedu.classify.logic.commands.ViewStatsCommand;
+import seedu.classify.logic.parser.exceptions.ParseException;
+import seedu.classify.model.student.Class;
+
+/**
+ * Parses input arguments and creates a ViewStatsCommand object
+ * */
+public class ViewStatsCommandParser implements Parser<ViewStatsCommand> {
+
+    @Override
+    public ViewStatsCommand parse(String args) throws ParseException {
+        Prefix filterPrefix = new Prefix("filter/");
```

Comment:
Ohh alright, wanted to do that originally but didn't since its not a prefix related directly to the student unlike the other prefixes, but I agree that including it there is better!